<launch>

    <arg name="camera_name" default="Logitech" />

    <node name="Logitech" pkg="camnode" type="webcamnode" output="screen" >
        <!--<remap from="/camnode/image_raw" to="/pose_estimator/image_raw" />-->
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="camera_frame_id" value="$(arg camera_name)" />
        <param name="camera_name" value="$(arg camera_name)" />
        <param name="camera_info_url" value="file:///home/jay/ros_workspace/src/camnode/camera_info/${NAME}.yaml" />
    </node>

    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen" >
        <remap from="image" to="/Logitech/image_raw" />
        <param name="autosize" value="true" />
    </node>

</launch>